terraform/paas/test.env.tfvars

```diff
@@ -5,3 +5,15 @@ paas_redis_1_name          = "get-into-teaching-test-redis-svc"
 paas_additional_route_name = "staging-getintoteaching"
 additional_routes          = 1
 
+alerts = {
+  GiT_App_Staging_Healthcheck = {
+    website_name  = "Get Into Teaching Website (Staging)"
+    website_url   = "https://get-into-teaching-app-test.london.cloudapps.digital/healthcheck.json"
+    test_type     = "HTTP"
+    check_rate    = 60
+    contact_group = [185037]
+    trigger_rate  = 0
+    custom_header = "{\"Content-Type\": \"application/x-www-form-urlencoded\"}"
+    status_codes  = "204, 205, 206, 303, 400, 401, 403, 404, 405, 406, 408, 410, 413, 444, 429, 494, 495, 496, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 521, 522, 523, 524, 520, 598, 599"
```

Comment:
Based off 

https://github.com/hashicorp/terraform-provider-statuscake/blob/stable-website/website/docs/r/test.html.markdown

This is just the default definition of `status_codes` - would it be better to just rely on upstreams default list rather than overriding it here?

Reply:
OK. if you like, ill make a change